### PR TITLE
Initial commit for cypressv6

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into Cypress require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/validators/checklist_criteria_validator.rb [hunk 1]
```diff
@@ -114,6 +123,11 @@ def verify_facility_location_attribute(attribute, checked_criteria)
       !attribute['code'].nil?
     end
 
+    def verify_entity_attribute(attribute)
+      # An quantity attribute should have (at a minimum) a 'value' value
```
Comment (lmd59) [inline lib/validators/checklist_criteria_validator.rb]: this code comment is not relevant

**Comment by lmd59** [comment]
Issues found for test created with CMS9v8:
Screen Shot 2020-02-07 at 9 44 50 AM
Screen Shot 2020-02-07 at 9 43 59 AM
Screen Shot 2020-02-07 at 9 43 28 AM
